### ISSUE TITLE
Python 3.6 compatibility

### ISSUE THREAD

**Issue by kielnino** [dimostenis/color-hash-python#3]
Hi, thanks for your work. I would really like to use it, but when I try to load the package on Python 3.6.9 I get:

`SyntaxError: future feature annotations is not defined`

In the Readme it says, its supported by Python 3.6+. A quick google-search revealed, that this feature is only supportet with Python 3.7. Maybe you should update the Readme?

**Comment by dimostenis** [comment]
Hello.
I tested it with tox against py3.6 (see pyproject.toml). Its really interesting. I will investigate it after weekend. Thanks for reporting it and for you patience.

**Comment by dimostenis** [comment]
It shall be resolved in [v1.2.0](https://github.com/dimostenis/color-hash-python/releases/tag/v1.2.0). Already published to pypi.

I used typing supported by 3.6. Even tested manually with py36 venv.

```bash
❯ tox -p
✔ OK python3.7 in 14.335 seconds
✔ OK python3.10 in 14.347 seconds
✔ OK python3.8 in 14.424 seconds
✔ OK python3.9 in 14.447 seconds
✔ OK python3.6 in 14.548 seconds
✔ OK pypy3.9 in 14.578 seconds
________________________ summary ________________________
  python3.6: commands succeeded
  python3.7: commands succeeded
  python3.8: commands succeeded
  python3.9: commands succeeded
  python3.10: commands succeeded
  pypy3.9: commands succeeded
  congratulations :)
```
